hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/metrics/MetricsContext.java

```diff
@@ -208,4 +213,51 @@ public long get() {
             return VOLATILE_VALUE_UPDATER.get(this);
         }
     }
+
+    private static final class DummyMetric implements Metric {
+
+        private final String name;
+        private final Unit unit;
+
+        DummyMetric(String name, Unit unit) {
+            this.name = name;
+            this.unit = unit;
+        }
+
+        @Nonnull
+        @Override
+        public String name() {
+            return name;
+        }
+
+        @Override
+        public Unit unit() {
+            return unit;
+        }
+
+        @Override
+        public void increment() {
+        }
+
+        @Override
+        public void increment(long amount) {
+        }
+
+        @Override
+        public void decrement() {
+        }
+
+        @Override
+        public void decrement(long amount) {
+        }
+
+        @Override
+        public void set(long newValue) {
+        }
+
+        @Override
+        public long get() {
+            return 0;
```

Comment:
This is wrong...

Reply:
Will fix by removing the `Metric.get()` method.